src/org/omegat/util/gui/LookAndFeelHelper.java

```diff
@@ -0,0 +1,113 @@
+/**************************************************************************
+ OmegaT - Computer Assisted Translation (CAT) tool
+          with fuzzy matching, translation memory, keyword search,
+          glossaries, and translation leveraging into updated projects.
+
+ Copyright (C) 2020 Hiroshi Miura
+               Home page: http://www.omegat.org/
+               Support center: https://omegat.org/support
+
+ This file is part of OmegaT.
+
+ OmegaT is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ OmegaT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ **************************************************************************/
+
+package org.omegat.util.gui;
+
+import javax.swing.UIManager;
+import java.awt.Color;
+
+import org.omegat.util.Log;
+
+/**
+ * Helper to initialize LaF.
+ */
+public class LookAndFeelHelper {
+
+    /** is LaF theme dark? */
+    private static Boolean isDark = null;
+
+    /**
+     * Heuristic detection of dark theme.
+     * <p>
+     *     isDarkTheme method derived from NetBeans licensed by Apache-2.0
+     *     and re-licensed into GPL3.
+     * @return true when dark theme, otherwise false.
+     */
+    public static boolean isDarkTheme() {
+        if(isDark != null) {
+            return isDark;
+        }
```

Comment:
I don't see a need for this to be a public API.

Then, since it's private and we only call it once, there's no need to cache the result.

```suggestion
    private static boolean isDarkTheme() {
```